jdaviz/configs/cubeviz/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -192,3 +192,42 @@ def test_save_collapsed_to_fits(cubeviz_helper, spectrum1d_cube_with_uncerts, tm
     with pytest.raises(ValueError, match="Invalid path=/this/path/doesnt"):
         extract_plugin._obj.vue_save_as_fits()
     extract_plugin._obj.filename == fname  # set back to original filename
+
+
+def test_aperture_markers(cubeviz_helper, spectrum1d_cube):
+
+    cubeviz_helper.load_data(spectrum1d_cube)
+    cubeviz_helper.load_regions([CirclePixelRegion(PixCoord(0.5, 0), radius=1.2)])
```

Comment:
Since this is new territory, you might also want to make sure:

1. Sky region is supported properly.
2. Have a real science use case where the target is off-center, so we know for sure the transposing of x/y/z isn't secretly messing things up.

Reply:
Before cone is even a thing, shouldn't we make sure a normal cylindrical use case even works properly? Or is this already tested elsewhere?

Reply:
As far as spectral extraction or the previews are concerned?  Cylinder spectral extraction existed before this, I'm not sure how well it is covered.  The previews have full line coverage, but obviously not too robustly with actual assertions... but its really just UI sugar on top of existing functionality to get the polygon.  If you have ideas for specific tests, feel free to push directly to this branch.

Reply:
Alas, I don't. OK we always have that "science verification" ticket buried somewhere in JIRA for the future...